scripts/dependency-graph-generator/index.js

```diff
@@ -0,0 +1,119 @@
+const graphviz = require('graphviz');
+const parser = require('dotparser');
+const readFileSync = require('fs').readFileSync;
+const spawnSync = require('child_process').spawnSync;
+const findGitRoot = require('../monorepo/index').findGitRoot;
+const path = require('path');
+
+const dotFilePath = path.resolve(__dirname, 'repo-graph.dot');
+
+function main() {
+  _generateGraphforRepo(dotFilePath);
+  const graph = _parseDotFile(dotFilePath);
+  const subTree = _getSubTree(graph, '@fluentui/react-button');
+
+  subTree.setGraphVizPath('/usr/bin');
```

Comment:
Is this safe for Windows?

Reply:
Good point, I need to make this a positional arg

Reply:
I added an arg for this, is that good enough ? or should I add a default windows path

Reply:
Added an error if the arg is not used with win32